sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -4049,7 +4049,7 @@ subroutine read_netcdf_3d_para(ncid,im,jm,ista,ista_2l,iend,iend_2u,jsta,jsta_2l
       implicit none
       INCLUDE "mpif.h"
 
-      character(len=20),intent(in) :: varname
+      character(len=*),intent(in) :: varname
```

Comment:
@SamuelTrahanNOAA Please add a change log with your name at the beginning session of this subroutine.

Reply:
The subroutine has no author information. Instead, all author information is at the top of the file. I added it there instead. Is that okay?

Reply:
@SamuelTrahanNOAA Yes, thanks for update.

Reply:
I left this change in by accident, but it is a bug fix. The character strings sent in are of length 4, not 20, which can produce an out-of-bounds read.